### ISSUE TITLE
XMLBuilder no longer calling tagValueProcessor function

### ISSUE THREAD

**Issue by rpastro** [NaturalIntelligence/fast-xml-parser#437]
* [x] Are you running the latest version?
* [x] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [ ] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)?

### Description
Starting with fast-xml-parser v4.0.2 the XMLbuilder is no longer calling the tagValueProcessor function.

### Code
```js
const { XMLBuilder } = require('fast-xml-parser');
const xmlBuilder = new XMLBuilder({
    processEntities: false,
    tagValueProcessor: (tagName, tagValue) => tagValue.toUpperCase()
});
xmlBuilder.build({ root: { test: 'hello' } });
```

### Output
With fast-xml-parser **v4.0.1**, the output is as expected:

`<root><test>HELLO</test></root>`

Starting with fast-xml-parser **v4.0.2**, the tagValueProcessor function is not being called, so the output is:

`<root><test>hello</test></root>`

### expected data
This is existing functionality which was broken by a minor release. Expect functionality is for tagValueProcessor to be invoked for all tags.

**Would you like to work on this issue?**

- [ ] Yes
- [ ] No
- [x] If Needed

**Comment by amitguptagwl** [comment]
Yes, I also doubt there. And by not having some missing tests it was not caught.

**Comment by amitguptagwl** [comment]
Please check v4.0.5

**Comment by rpastro** [comment]
@amitguptagwl   This problem has not been solved with v4.0.5.  Can you please reopen this issue?

**Comment by amitguptagwl** [comment]
Thanks @rpastro . It is fixed now. Releasing v4.0.6

**Comment by Elity** [comment]
@amitguptagwl XMLParse not calling tagValueProcessor function
